master/getting-started/kubernetes/installation/calico.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Installing Calico on Kubernetes with Calico networking
+canonical_url: 'https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/calico'
+---
+
+## Before you begin
+
+- Ensure that you have a Kubernetes cluster that meets the 
+  {{site.prodname}} [system requirements](../../requirements). If you don't, 
+  follow the steps in [Using kubeadm to create a cluster](http://kubernetes.io/docs/getting-started-guides/kubeadm/) to create one.
+
+- If your cluster has RBAC enabled, issue the following command to 
+  configure the roles and bindings that {{site.prodname}} requires.
+
+   ```
+   kubectl apply -f \
+   {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml
+   ```
+   > **Note**: You can also 
+   > [view the manifest in your browser](hosted/rbac-kdd.yaml){:target="_blank"}.
+   {: .alert .alert-info}
+   
+## Installing {{site.prodname}} with {{site.prodname}} networking
+
+### About installing {{site.prodname}} with {{site.prodname}} networking
+
+The procedure differs according to the type of datastore you want {{site.prodname}}
+to use and the number of nodes. Refer to the section that matches your configuration.
```

Comment:
I think we're assuming that people coming here are going to be installing Calico for the first time, so "matches your configuration" doesn't make sense for them, since this is where they'll pick their configuration.

Reply:
To that end, I feel like we really want some sort of recommendation between the two data stores to make this easier for users, or at least a way to let users know what the differences are here and why the should chose one over the other.

Maybe we need a brief section called "Choosing a data store" which covers this?